spec/lib/decidim/vocdoni/module_config_spec.rb

```diff
@@ -42,6 +42,12 @@ def cmd_capture(cmd, env: {})
         expect(endpoint_env.strip).to eq("stg")
       end
 
+      it "has the correct endpoint url" do
```

Comment:
This is for another PR/issue/discussion:

Having the hardcoded URLs in the module was because we didn't have a way of knowing from Ruby-land the values from the NPM package. Now that we have the SDK integrated in Ruby we could extract it with the node-wrapper, right @microstudi?

Reply:
Moved to issue https://github.com/decidim-vocdoni/decidim-module-vocdoni/issues/79